### ISSUE TITLE
Symfony 4 support (3.4.9 deprecation warning)

### ISSUE THREAD

**Issue by nmeirik** [winzou/StateMachineBundle#46]
[Version 0.3.1](https://github.com/winzou/StateMachineBundle/releases/tag/v0.3.1) claims to be Symfony4 compatible, but when running this version as a part of Symfony 3.4.9, I still get the following deprecation warning:

```
Auto-registration of the command "winzou\Bundle\StateMachineBundle\Command\winzouStateMachineDebugCommand" is deprecated since Symfony 3.4 and won't be supported in 4.0. Use PSR-4 based service discovery instead.
```

**Comment by winzou** [comment]
Fixed by #43 ;)

**Comment by nmeirik** [comment]
Great news! Will there be a new release containing the fix?
